src/data-access/repositories/interfaces/piece-repository.ts

```diff
@@ -5,6 +5,7 @@ export interface PieceRepository {
   getPiecesFromIds(pieceIds: string[]): Promise<Piece[]>
   savePiece(piece: Piece): Promise<void>
   deletePiecesForPart(partId: string): Promise<void>
+  deletePieces(pieceIdsToBeDeleted: string[]): Promise<void>
```

Comment:
When the function name is `deletePieces`, is it imperative that the parameter name be `pieceIdsToBeDeleted`. Isn't it clear from the function name that `pieceIds` will be deleted.

Reply:
I would agree. I keep going back and forth on it. It's been updated :)